### PR TITLE
Base.ProviderIdentification Validation

### PR DESCRIPTION
**Pull request by oajimenez**
Changed select statement to lateral flatten format, deleted window function, and now everything is within margin. 

Performed sample validation with composite PK of ["IDENTIFICATIONVALUE", "IDENTIFICATIONTYPEID"].

### CODE REVIEW

File: migration_original/ODS1Stage/tables/Base/ProviderIdentification/spu_translated_ProviderIdentification.sql [hunk 1]
```diff
@@ -35,25 +36,31 @@ begin
...
+                        p.providerid,
+                        i.identificationtypeid,
+                        json.identification_Identifier as IdentificationValue,
+                        json.identification_ExpirationDate as ExpirationDate,
+                        json.identification_SourceCode as SourceCode,
+                        json.identification_LastUpdateDate as LastUpdateDate
+                    from cte_identification as json
+                    join base.provider as p on p.providercode = json.providercode
+                    join base.identificationtype as i on i.identificationtypecode = json.identification_IdentificationTypeCode
+                    where identificationtypeid is not null
```
Comment (carresemata) [inline migration_original/ODS1Stage/tables/Base/ProviderIdentification/spu_translated_ProviderIdentification.sql:62]: not necessary this where clause

File: migration_original/ODS1Stage/tables/Base/ProviderIdentification/spu_translated_ProviderIdentification.sql [hunk 2]
```diff
@@ -35,25 +36,31 @@ begin
...
+                    $$;
```
Comment (carresemata) [inline migration_original/ODS1Stage/tables/Base/ProviderIdentification/spu_translated_ProviderIdentification.sql:63]: in the merge add the identificationtypeid as well in the join

**Review by carresemata — CHANGES_REQUESTED** [review]

File: migration_original/ODS1Stage/tables/Base/ProviderIdentification/spu_translated_ProviderIdentification.sql (hunk 2 above)
Comment (oajimenez) [reply migration_original/ODS1Stage/tables/Base/ProviderIdentification/spu_translated_ProviderIdentification.sql:62]: ✅

**Review by carresemata — APPROVED** [review]
GTG

**Pull request merged**